README.adoc

```diff
@@ -3,164 +3,159 @@
+./gradlew bootRun
 ----
 ====
 
-include::https://raw.githubusercontent.com/spring-guides/getting-started-macros/main/spring-boot-application-new-path.adoc[]
+This will start the application and the method annotated with `@Scheduled` will execute. You will see log messages similar to:
 
-The
-https://docs.spring.io/spring/docs/current/spring-framework-reference/htmlsingle/#scheduling-enable-annotation-support[`@EnableScheduling`]
-annotation ensures that a background task executor is created. Without it, nothing gets
-scheduled.
+====
+----
+20yy-mm-ddT07:23:01.665-04:00  INFO 19633 --- [   scheduling-1] c.e.schedulingtasks.ScheduledTasks       : The time is now 07:23:01
+20yy-mm-ddT07:23:06.663-04:00  INFO 19633 --- [   scheduling-1] c.e.schedulingtasks.ScheduledTasks       : The time is now 07:23:06
+20yy-mm-ddT07:23:11.663-04:00  INFO 19633 --- [   scheduling-1] c.e.schedulingtasks.ScheduledTasks       : The time is now 07:23:11
+----
+====
 
-include::https://raw.githubusercontent.com/spring-guides/getting-started-macros/main/build_an_executable_jar_subhead.adoc[]
+NOTE: This example uses https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/scheduling/annotation/Scheduled.html#fixedRate()[`fixedRate()`^] scheduling, so the application will run indefinitely until you interrupt the execution manually.
 
-include::https://raw.githubusercontent.com/spring-guides/getting-started-macros/main/build_an_executable_jar_with_both.adoc[]
+== Testing with the `awaitility` Dependency
 
-Logging output is displayed, and you can see from the logs that it is on a background
-thread. You should see your scheduled task fire every five seconds. The following listing
-shows typical output:
+In order to properly test your application, you can use the https://github.com/awaitility/awaitility[`awaitility` library^]. Since Spring Boot 3.2, this is a dependency that Boot manages. Create a new test or view the existing test at `src/test/java/com/example/schedulingtasks/ScheduledTasksTest.java`
 
 ====
-[source,text]
+[source,java]
 ----
-...
-2019-10-02 12:07:35.659  INFO 28617 --- [   scheduling-1] c.e.schedulingtasks.ScheduledTasks       : The time is now 12:07:35
-2019-10-02 12:07:40.659  INFO 28617 --- [   scheduling-1] c.e.schedulingtasks.ScheduledTasks       : The time is now 12:07:40
-2019-10-02 12:07:45.659  INFO 28617 --- [   scheduling-1] c.e.schedulingtasks.ScheduledTasks       : The time is now 12:07:45
-2019-10-02 12:07:50.657  INFO 28617 --- [   scheduling-1] c.e.schedulingtasks.ScheduledTasks       : The time is now 12:07:50
-...
+@SpringBootTest
+public class ScheduledTasksTest {
+
+	@SpyBean
+	ScheduledTasks tasks;
+
+	@Test
+	public void reportCurrentTime() {
+		await().atMost(Durations.TEN_SECONDS).untilAsserted(() -> {
+			verify(tasks, atLeast(2)).reportCurrentTime();
+		});
+	}
+}
 ----
 ====
 
+This test will automatically be executed when running the task `./gradlew clean build`.
```

Comment:
Change to:

This test automatically runs when you run the `./gradlew clean build` task.